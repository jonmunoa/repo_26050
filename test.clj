(defn hello-world []
  (println "Hello, World!"))

(hello-world)(defn hello-world []
  (println "Hello, World!"))

(hello-world)(defn hello-world []
  (println "Hello, World!"))

(hello-world)(defn hello-world []
  (println "Hello, World!"))

(hello-world)(defn hello-world []
  (println "Hello, World!"))

(hello-world)
