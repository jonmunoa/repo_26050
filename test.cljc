#?(:clj
  (defn hello-world []
    (println "Hello, Clojure!"))
  (hello-world))
#?(:cljs
  (js/console.log "Hello, ClojureScript!"))
#?(:clj
  (defn hello-world []
    (println "Hello, Clojure!"))
  (hello-world))
#?(:cljs
  (js/console.log "Hello, ClojureScript!"))
#?(:clj
  (defn hello-world []
    (println "Hello, Clojure!"))
  (hello-world))
#?(:cljs
  (js/console.log "Hello, ClojureScript!"))
#?(:clj
  (defn hello-world []
    (println "Hello, Clojure!"))
  (hello-world))
#?(:cljs
  (js/console.log "Hello, ClojureScript!"))
