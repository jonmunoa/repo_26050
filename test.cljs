(ns hello-world.core
  (:require [cljs.core :as cljs]))

(cljs/println "Hello, ClojureScript!")
(ns hello-world.core
  (:require [cljs.core :as cljs]))

(cljs/println "Hello, ClojureScript!")
(ns hello-world.core
  (:require [cljs.core :as cljs]))

(cljs/println "Hello, ClojureScript!")
(ns hello-world.core
  (:require [cljs.core :as cljs]))

(cljs/println "Hello, ClojureScript!")
(ns hello-world.core
  (:require [cljs.core :as cljs]))

(cljs/println "Hello, ClojureScript!")
