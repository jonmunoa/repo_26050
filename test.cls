Public Class Persona
    Public Nombre As String
    Public Edad As Integer
End Class
Public Class Persona
    Public Nombre As String
    Public Edad As Integer
End Class
Public Class Persona
    Public Nombre As String
    Public Edad As Integer
End Class
Public Class Persona
    Public Nombre As String
    Public Edad As Integer
End Class
Public Class Persona
    Public Nombre As String
    Public Edad As Integer
End Class
Public Class Persona
    Public Nombre As String
    Public Edad As Integer
End Class
Public Class Persona
    Public Nombre As String
    Public Edad As Integer
End Class
Public Class Persona
    Public Nombre As String
    Public Edad As Integer
End Class
Public Class Persona
    Public Nombre As String
    Public Edad As Integer
End Class
