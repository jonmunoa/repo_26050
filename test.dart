void main() {
  print('Hello, Dart!');
}
void main() {
  print('Hello, Dart!');
}
void main() {
  print('Hello, Dart!');
}
void main() {
  print('Hello, Dart!');
}
void main() {
  print('Hello, Dart!');
}
void main() {
  print('Hello, Dart!');
}
void main() {
  print('Hello, Dart!');
}
void main() {
  print('Hello, Dart!');
}
