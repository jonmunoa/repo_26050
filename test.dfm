object MainForm: TForm
  Caption = 'Mi aplicación'
  ClientHeight = 300
  ClientWidth = 400
  object Label1: TLabel
    Left = 100
    Top = 100
    Caption = 'Hola Mundo!'
  end
end
object MainForm: TForm
  Caption = 'Mi aplicación'
  ClientHeight = 300
  ClientWidth = 400
  object Label1: TLabel
    Left = 100
    Top = 100
    Caption = 'Hola Mundo!'
  end
end
object MainForm: TForm
  Caption = 'Mi aplicación'
  ClientHeight = 300
  ClientWidth = 400
  object Label1: TLabel
    Left = 100
    Top = 100
    Caption = 'Hola Mundo!'
  end
end
