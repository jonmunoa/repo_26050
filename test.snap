name: my-app
version: '1.0'
summary: Una breve descripción de mi aplicación.
description: |
  Una descripción más larga de mi aplicación.
  Puede incluir varias líneas de texto.

grade: stable
confinement: strict

apps:
  my-app:
    command: my-app
    plugs: [network, home]
